soc/arm/silabs_exx32/efr32bg22/Kconfig.defconfig.series

```diff
@@ -18,6 +18,7 @@ config NUM_IRQS
 config PM
 	default n
 	select COUNTER
+	select UART_INTERRUPT_DRIVEN
```

Comment:
since PM has been dropped, you can likely drop this for now

Reply:
Removed this